fuel/app/classes/controller/welcome.php

```diff
@@ -16,42 +17,38 @@
  * A basic controller example.  Has examples of how to set the
  * response body and status.
  *
- * @package  app
  * @extends  Controller
  */
-class Controller_Welcome extends Controller
+class controller_welcome extends Controller
 {
-	/**
-	 * The basic welcome message
-	 *
-	 * @access  public
-	 * @return  Response
-	 */
-	public function action_index()
-	{
-		return Response::forge(View::forge('welcome/index'));
-	}
+    /**
+     * The basic welcome message.
+     *
+     * @return Response
+     */
+    public function action_index()
+    {
+        return Response::forge(View::forge('welcome/index'));
+    }
 
-	/**
-	 * A typical "Hello, Bob!" type example.  This uses a Presenter to
-	 * show how to use them.
-	 *
-	 * @access  public
-	 * @return  Response
-	 */
-	public function action_hello()
-	{
-		return Response::forge(Presenter::forge('welcome/hello'));
-	}
+    /**
+     * A typical "Hello, Bob!" type example.  This uses a Presenter to
+     * show how to use them.
+     *
+     * @return Response
+     */
+    public function action_hello()
+    {
+        return Response::forge(Presenter::forge('welcome/hello'));
```

Comment:
__[phpmd]__
* Avoid using static access to class 'Response' in method 'action_hello'.
* Avoid using static access to class 'Presenter' in method 'action_hello'.